docs/docs/getting-started/demo/without-docker.md

```diff
@@ -43,6 +43,19 @@ cabal exec cardano-node -- run \
 </TerminalWindow>
 ````
 
+## Seeding The Network
+
+You can use the `seed-devnet.sh` script by passing it the path to a cardano-cli executable to use, instead of having it using the Docker container. For example:
+
+
+```mdx-code-block
+<TerminalWindow>
+./seed-devnet.sh $(which cardano-cli)
```

Comment:
@KtorZ @ffakenz This will still use the `docker run .. hydra-node publish-scripts`

Reply:
Ah! Good point.

Reply:
Fixed it on the plane yesterday https://github.com/input-output-hk/hydra-poc/pull/459